Applications/Core TLP.md

```diff
@@ -0,0 +1,69 @@
+## Introduction
+
+Currently the Node.js Foundation directly oversees the development of the core platform and associated working groups. The addition of Top Level Projects will predictably increase the workload on the TSC and so it makes sense to transfer these responsibilities to a Top Level Project concerned solely with the development of the platform.
```

Comment:
This line is going to become redundant as soon as this is adopted. Perhaps this belongs under "History" or "Context" or something